### DISCUSSION TITLE
Ferramentas para organização

### DISCUSSION THREAD

**Discussion by mksanti** [frontendbr/forum#61] [Perguntas e Respostas]
Fala galera, ontem vendo a issue #54 o @rfviolato comentou que utiliza o [Zube](https://zube.io/) (curti esta ferramenta) para organizar issues. 
To abrindo esta issue para gente dicutir sobre as ferramentas que vocês utilizam para organizar os projetos.

Eu particularmente utilizo o proprio Github e para projetos pessoais utilizo [Wunderlist](http://wunderlist.com/) + [Toggl](http://toggl.com/) e vcs ?

**Comment by rubenmarcus** [comment]
Wriker

Zappier infelizmente é pago,mas é muito bom pra integrações de Apps.

Nas empresas q trabalhei tinha TFS + Git. Gostava das partes de Agile e Scrum, ver os sprints e etc. Não achei nenhuma Free q integrasse o Git. Eu sei que o Wriker tem ate Gant Chart mas ai é pago.

**Comment by felipefialho** [comment]
Atualmente estamos usando o [Jira](https://www.atlassian.com/software/jira), é muito completo.

Como já usamos várias ferramentas da Atlassian, como o BitBucket, a integração é perfeita.

Recomendo.

**Comment by yuricamara** [comment]
Estou usando o Zoho. A versão free já me satisfaz.  Mas usaria o Jira se não fosse pelo valor.

**Comment by karlprieb** [comment]
Aqui usávamos o WorkFront, mas é bem ruinzinho. No meio do projeto migramos para o Jira :) Bem completo e fácil de integrar com qualquer workflow.

**Comment by vitorbritto** [comment]
@LFeh as ferramentas da Atlassian são show de bola! Fiz um teste com o Workflow contendo: Jira + HipChat + BitBucket + SourceTree + Bamboo. Não tenho do que reclamar. Mais para frente vou montar esse workflow.

Entretanto, utilizo no momento: 
- Trello (para tarefas com um _mix_ de GTD e Scrum)
- BitBucket/GitHub Issues (para issues tracking dos projetos)
- Evernote (para organizar meus pensamentos)
- Post-It (uso muito, mas muito mesmo)
- Lousa + Pilot

**Comment by felipefialho** [comment]
@vitorbritto Inclusive o HipChat temos utilizado bastante, mas ainda estamos em dúvida entre ele e o Slack.

**Comment by rickbenetti** [comment]
Aqui utilizamos:
- Trello (para gestão como Kanban, mas com mais colunas a depender do projeto)
- Gitlab (vamos migrar do Trello todas as issues para melhor trackeamento do que é feito)
- Office 365 (temos alguns pptx, xlsx principalmente de clientes, docx de propostas, contratos e tals)

Notas:
- Já usamos o hipchat, mas no fim das contas sempre voltamos pro Skype que é possível fazer compartilhamento de tela e falar com o cliente e equipe no mesmo aplicativo e o hipchat é um chat com integração nada mais;
- Já tentamos o slack, mas novamente sempre voltamos pro Skype.

**Comment by felipefialho** [comment]
@rkb81 Para conversas em grupos ou uma simples call, também usamos o Skype.

**Comment by vitorbritto** [comment]
@LFeh ambos são excelentes, não utilizo em todos os projetos pois só vejo sentido em usá-los quando se trata de um _job_ de  médio-grande porte e necessita de uma interação contínua dos _stakeholders_. =]

**Comment by tcelestino** [comment]
Slack + Trello.

**Comment by lucasbento** [comment]
Eu uso Slack, Trello, Bitbucket e quando preciso fazer uma vídeo-chamada uso [appear.in](https://appear.in/) que é bem simples ou então o Hangouts. Também usava o Skype antes porém ele tem um jeitinho mágico de fazer meu note travar todo.

@rubenmarcus Na minha empresa também utilizamos o Zapier, realmente tem muitas integrações úteis e na verdade ele tem um plano free, porém com integrações e quantidade de execuções no mês bem baixo. Recomendo o [IFTTT](https://ifttt.com/), não é um Zapier mas dá pra usar numa boa.

**Comment by felipefialho** [comment]
Legal esse appear.in. Não conhecia. 

O Skype tem muitas vantagens, mas o tanto que ele é pesado complica as coisas as vezes.

**Comment by cleberoliveira** [comment]
Na empresa utilizo Jira + Slack e para projetos freelas que faço com a galera utilizo Slack + Asana. Antes eu utilizava o Trello, porém como lido com diversos projetos / clientes, achei melhor o Asana... no fim eu estava ficando com muitos boards.

De todos o Jira para mim é muito bom, porém o preço não ajuda :P

**Comment by lucasbento** [comment]
Eu usei o JIRA por um tempo, também curti bastante, realmente tem tudo que precisamos mas a receita da empresa preferiu o Trello, nele só sinto falta de um export das tasks descente, algo que aparentemente só tem pagando.

**Comment by felipefialho** [comment]
E a integração do Jira com o Bitbucket é realmente impressionante. 

Porém várias coisas realmente são pagas @lucasbsilva, mas talvez vale comentar com sua empresa sobre isso, a ferramenta é muito boa.

**Comment by lucasbento** [comment]
@LFeh valeu pelo conselho, vou tentar isso daqui um tempo, agora a receita tá meio curta e é compreensível cancelar alguns serviços, estamos _mais ou menos_ conseguindo nos virar com o Trello.

**Comment by rickbenetti** [comment]
@lucasbsilva o Jira apesar de ser pago ele tem uma versão que custa $10 para equipes pequenas o que vale muito a pena.

**Comment by lucasbento** [comment]
Interessante, @rkb81, não sabia disso, com certeza vou correr atrás.

**Comment by felquis** [comment]
Na Pagar.me usamos muito o Trello, o GitHub, Zendesk, Slack e uma maçaroca de ferramentas, porém o preço dessas ferramentas em uma empresa com muitas pessoas, quando tem que pagar por quantidade de funcionários ainda por cima é muito caro.

Estou com uma plano maligno hahaha de colocar o máximo de pessoas que eu conseguir dentro do GitHub para manter tracking de tudo que é reportado em issues, e usar o appearin/discord para várias reuniões remotas. Estou me baseando muito na [cultura do GitLab](https://about.gitlab.com/handbook/#communication) e estou pregando muito a cultura Open Source dos devs para todos.

Esses dias eu fiz a seguinte continha:

```js 
function shouldIUseThisTool() {
  const oneDollarInBrazilianReal = 312
  const pricePerMonthInDollar = 1250
  const collaborators = 200

  const oneYear = 12
  const pricePerColaboratorInAYear = pricePerMonthInDollar * oneYear
  const priceForWholeCompanyInAYear = (
    collaborators * pricePerColaboratorInAYear
  )
  const priceForWholeCompanyInAYearInReal = (
    (priceForWholeCompanyInAYear * oneDollarInBrazilianReal) / 100
  )

  return priceForWholeCompanyInAYearInReal
}
```
